frontend/src/pages/LearnPage/TeachersTips/InFrontOfChildren.tsx

```diff
@@ -0,0 +1,78 @@
+import { Section, SectionTitle, List, ListItem, Paragraph } from './styled';
+import BackToTopButton from './BackToTopButton';
+import { Text } from './NotAboutTeaching';
+
+const data: Text[] = [
+  {
```

Comment:
tyhle celý data bych nadefinoval jinak.

Stačí to takhle 
```tsx
const data: JSX.Element[] = [
  <>
     Dej si <strong>dost času na seznámení s dětmi</strong>, není kam spěchat ...
  </>,
  <>
    Nastav si s nimi <strong>jasná pravidla</strong>, ať ví, co od tebe mohou čekat.
  </>
   ...
]
```

Takže do toho pole dáš rovnou i ten HTML kód